integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthenticatorsIT.groovy

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.resource.authenticator.Authenticator
+import com.okta.sdk.resource.authenticator.AuthenticatorList
+import com.okta.sdk.resource.authenticator.AuthenticatorStatus
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.annotations.Test
+
+import java.util.stream.Collectors
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+/**
+ * Tests for {@code /api/v1/authenticators}.
+ * These tests requires the Org to be OIE enabled.
+ * @since 6.x.x
+ */
+class AuthenticatorsIT extends ITSupport {
+
+    @Test (groups = "bacon")
```

Comment:
Add a comment pointing to OKTA-433369 (note that it'll be enabled when that ticket is done).